mobile/src/main/java/org/openhab/habdroid/ui/WidgetAdapter.kt

```diff
@@ -1702,6 +1708,29 @@ class WidgetAdapter(
         }
     }
 
+    class ColorTemperatureViewHolder internal constructor(initData: ViewHolderInitData) : LabeledItemBaseViewHolder(
+        initData,
+        R.layout.widgetlist_colortemperatureitem,
+        R.layout.widgetlist_colortemperatureitem_compact
+    ) {
+        private val previewImage = itemView.findViewById<ImageView>(R.id.current_temperature)
+
+        override fun bind(widget: Widget) {
+            super.bind(widget)
+            val drawable = (widget.state ?: widget.item?.state)?.asNumber?.let { state ->
+                // if not Kelvin, assume Mirek
+                val kelvin = if (state.unit == "K") state.value else 1000000F / state.value
```

Comment:
Alternative logic mentioned [here](https://github.com/openhab/openhab-core/issues/3891#issuecomment-2509668785) ... not sure what I prefer yet.

Reply:
I've implemented the distinction by value (< 1000 is Mirek / > 1000 is Kelvin) for now.